core/src/main/scala/kafka/controller/KafkaController.scala

```diff
@@ -1472,6 +1525,49 @@ class KafkaController(val config: KafkaConfig, zkUtils: ZkUtils, val brokerState
     }
   }
 
+  case class LeaderAndIsrResponseReceived(LeaderAndIsrResponseObj: AbstractResponse, brokerId: Int) extends ControllerEvent {
```

Comment:
Can we make this naming consistent with TopicDeletionStopReplicaResult?

We can either change the other class name or maybe change your new class to something like LeaderAndIsrResult or LeaderAndIsrResponseResult?